### PR TITLE
Move constants to common for clean dependency graph.

### PR DESCRIPTION
**Pull request by travis-minke-sap**
The KafkaChannel `pkg/apis/` content has a couple of dependencies into the `pkg/channel/consolidated/` implementation which results in a circular dependency chain.  This is not causing a problem (build or otherwise), but is just generally wrong and should be de-tangled.  It's only a couple constant values that need to be moved - just basic cleanup : )

### CODE REVIEW

File: pkg/common/constants/constants.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package constants
```
Comment (matzew) [inline pkg/common/constants/constants.go:1]: let's add the license header

Comment (travis-minke-sap) [reply pkg/common/constants/constants.go:1]: agreed - will have to get used to that (setup my intellij file creation template ; )

**Comment by travis-minke-sap** [comment]
/test pull-knative-sandbox-eventing-kafka-integration-tests

grrr... flaky integration tests...

**Pull request merged**